configuration/pih/drugs/drugs.csv

```diff
@@ -124,6 +124,7 @@ c155593e-12f3-42aa-a592-25a1cad7bbb7,10561,"Ephedrine hydrochloride, Solution fo
 b3df9e4b-3cd0-4412-a250-4c4e6783d3c2,11356,"Erythromycin, 500mg film coated tablet",CIEL:75842,
 638d29ff-96fa-4f6e-8d23-0c92f43d5d1a,11866,"Ethambutol (E), 100mg tablet",PIH:745,
 bd159878-7a33-405b-ad0b-6d6ddbf99ddd,11849,"Ethambutol (E), 400mg tablet",PIH:745,
+0f3b5757-30ab-46cd-8e59-02cb88ae3e1f,,"Ethambutol + Isoniazide 400/150 mg - 550 milligram Tablet",PIH:1108,
```

Comment:
Awaiting OpenBoxes code, @lnball is it okay to leave blank for now?
Also, please tweak name as you see fit